### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by Herrscher-of-Origin** [share121/inter-knot#4910] [委托]
我的保险箱在克里特空洞里丢了，有没有人能帮我找回来，是一个大概50*50*100的保险箱，酬金可以详谈。

**Comment by shidijun** [comment]
这个委托我有能力接 但是首先得先了解一下具体详情 顺便谈一下酬金

**Comment by BJ-cmyk** [comment]
这里总算没有人挂脚本了吧？💀

**Comment by 1luik** [comment]
你可以试试在丢的附近用强磁铁试试

**Comment by cuijiaqi0406** [comment]
这个委托我接了！<img src="https://bbs-static.miyoushe.com/static/2024/03/21/f5f9f9902140f168d5b804fe6c365ea6_4229255832036354153.png" alt="丽都漫步-哲" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/03/21/f5f9f9902140f168d5b804fe6c365ea6_4229255832036354153.png" alt="丽都漫步-哲" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/03/21/f5f9f9902140f168d5b804fe6c365ea6_4229255832036354153.png" alt="丽都漫步-哲" width="75" >

**Comment by KELERSAMA** [comment]
是天马速运吗？天马速运就是差劲辣（去掉括号，一条2丁尼）
